src/scalars/time/GraphQLDate.js

```diff
@@ -33,7 +42,7 @@ export default new GraphQLScalarType({
         if (typeof value === 'string' || value instanceof String) {
             validateDate(value);
             if (!isNaN(Date.parse(value))) {
-                return new Date(value);
+                return moment(value).format("YYYY-MM-DD[T]HH:mm:ss.SSS[Z]");
```

Comment:
Have you tought about extracting this into a constant?

Reply:
I guess it will be better, if it is a constant. Will change it.